pkg/cmd/packaging.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2019 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+func completionCmd(rootCmd *cobra.Command) *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "completion <bash|zsh>",
+		Short: "Generates completion scripts for bash and zsh",
+		Long: `To load completion run into your current shell run
+
+. <(kubeone bashCompletion <shell>)
+`,
+		Example:   "kubeone completion bash",
+		ValidArgs: []string{"bash", "zsh"},
+		Args:      cobra.ExactValidArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			switch args[0] {
+			case "bash":
+				err = rootCmd.GenBashCompletion(os.Stdout)
+			case "zsh":
+				err = rootCmd.GenZshCompletion(os.Stdout)
+			}
+			return
+		},
+	}
+	return cmd
+}
+
+func documentCmd(rootCmd *cobra.Command) *cobra.Command {
+	var path string
+	var cmd = &cobra.Command{
+		Use:       "document <man|md|rest|yaml>",
+		Short:     "Generates documentation",
+		Long:      "Generation can be generated as Man Pages, Markdown, ReStructured Text Docs or Yaml",
```

Comment:
I would replace `Generation` with `Documentation` and maybe put everything in lower case:

```suggestion
		Long:      "Documentation can be generated as man pages, markdown, ReStructured Text Docs or yaml",
```